packages/quantic/force-app/main/default/lwc/quanticSmartSnippet/e2e/quanticSmartSnippet.e2e.ts

```diff
@@ -0,0 +1,126 @@
+import {testSearch, testInsight} from './fixture';
+import {useCaseTestCases} from '../../../../../../playwright/utils/useCase';
+
+const maximumSnippetHeight = 250;
+
+const fixtures = {
+  search: testSearch,
+  insight: testInsight,
+};
+
+useCaseTestCases.forEach((useCase) => {
+  let test = fixtures[useCase.value];
+
+  test.describe(`quantic smart snippet ${useCase.label}`, () => {
+    test.use({
+      options: {maximumSnippetHeight},
+    });
+
+    test.describe('when the smart snippet is expanded', () => {
```

Comment:
Since most of the tests are testing the analytics anyway, I think we could simplify the structure of the file:

```
  test.describe('Smart Snippet analytics', () => {
    test.describe('when the smart snippet is expanded', () => {
      test('should send the source title clicked analytics',);
      test('should send the source clicked analytics',);
      test('should send the inline link clicked analytics',);
    });
    test('should send the expand and collapse analytics',);
    [...]
  });
```
Do you see what I mean? Could be very simple that way.
Other than that very clean tests :+1:

Reply:
Yeah maybe I mean the feedback ones are also analytics so at this point they all are analytics related :D